tc_aws/aws/bucket.py

```diff
@@ -0,0 +1,91 @@
+# coding: utf-8
+
+from tornado_botocore import Botocore
+import botocore.session
+from tornado.concurrent import return_future
+
+class Bucket(object):
+    _bucket = None
+
+    def __init__(self, bucket):
+        self._bucket = bucket
+
+    @return_future
+    def get(self, path, callback=None):
+        """
+        Returns object at given path
+        :param path:
+        :param callback:
+        """
+        session = Botocore(service='s3', region_name='eu-west-1', operation='GetObject')
+        session.call(
+            callback=callback,
+            Bucket=self._bucket,
+            Key=path,
+        )
+
+    @return_future
+    def get_url(self, key, method='GET', expiry=3600, callback=None):
+        """
+        Generates the presigned url for given key & methods
+        :param key:
+        :param method:
+        :param expiry:
+        :param callback:
+        :return:
+        """
+        session = botocore.session.get_session()
```

Comment:
Is there a reason in some places `session` is used and in other `Botocore(...)` ?